apps/server/src/modules/common-cartridge/common-cartridge.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common';
+import { FilesStorageModule } from '../files-storage/files-storage.module';
+import { CommonCartridgeExportService } from './service/common-cartridge-export.service';
+import { CommonCartridgeUc } from './uc/common-cartridge.uc';
+
+@Module({
+	imports: [FilesStorageModule],

```

Comment:
Replace this with FileStorageClientModule.

You are currently using the internal Service, which should only be called by API or AMQP.

The Client Module offers the same Methods so it should be mostly in-place